### PR TITLE
update colorscheme to colorblind-friendly Tol palette

### PR DESCRIPTION
**Pull request by mikemorris**
See https://davidmathlogic.com/colorblind/#%23332288-%23117733-%2344AA99-%2388CCEE-%23DDCC77-%23CC6677-%23AA4499-%23882255-%23808080 to view palette and for more information.

Screenshot 2024-03-25 at 11 27 05 AM

### CODE REVIEW

File: assets/html/index.html [hunk 1]
```diff
@@ -570,6 +570,18 @@ <H1>Faces</H1>
 		"yay":       "&#x1F389;",
 	};
 
+	static colors = {
+		"grey": "grey",
+		"purple": "rgb(48 34 130)",
+		"green": "rgb(55 117 59)",
+		"cyan": "rgb(55 117 59)", // too similar to grey, avoid
```
Comment (mikemorris) [inline assets/html/index.html:577]:
```suggestion
		"cyan": "rgb(55 117 59)", // too similar to grey and pink, avoid
```

File: pkg/faces/constants.go [hunk 2]
```diff
@@ -28,18 +28,31 @@ var Smileys = map[string]string{
 	"Screaming":   "&#x1F631;",
 }
 
+// colorblind-friendly colors from the Tol palette
+var Colors = map[string]string{
+	"grey":    "grey",
+	"purple":  "rgb(48 34 130)",
+	"green":   "rgb(55 117 59)",
+	"teal":    "rgb(55 117 59)",   // too similar to grey, avoid
```
Comment (mikemorris) [inline pkg/faces/constants.go:36]:
```suggestion
	"cyan":    "rgb(55 117 59)",   // too similar to pink and grey, avoid
```

**Comment by kflynn** [comment]
(That's the "Tol palette" as in "the color palette attributed to [Paul Tol](https://personal.sron.nl/~pault/#sec:colour_blindness) in [David Nichol's colorblindness page](https://davidmathlogic.com/colorblind/#%23332288-%23117733-%2344AA99-%2388CCEE-%23DDCC77-%23CC6677-%23AA4499-%23882255-%23808080). 🙂)

This is a great idea. We should probably go ahead and give the colors semantic names in the CSS, etc. -- I'll take a crack at that.

**Comment by kflynn** [comment]
(I'm going to go ahead and land this immediately, though.)

**Pull request merged**